diff --git a/node_modules/@khanacademy/react-multi-select/dist/dropdown.js b/node_modules/@khanacademy/react-multi-select/dist/dropdown.js
index cef09e5..a66fc5a 100644
--- a/node_modules/@khanacademy/react-multi-select/dist/dropdown.js
+++ b/node_modules/@khanacademy/react-multi-select/dist/dropdown.js
@@ -62,7 +62,6 @@ var Dropdown = function (_Component) {
                     _this.toggleExpanded(false);
                     break;
                 case 13: // Enter Key
-                case 32: // Space
                 case 40:
                     // Down Arrow
                     _this.toggleExpanded(true);
